config/runtime.exs

```diff
@@ -60,11 +60,14 @@ if sentry_dsn not in [nil, ""] do
     environment_name: env
 end
 
-scheduler_jobs =
-  if env == "prod",
-    do: [{"0 7 * * *", {Screenplay.Jobs.TakeoverToolTestingJob, :run, []}}],
-    else: []
-
-config :screenplay, Screenplay.Scheduler, jobs: scheduler_jobs
+if env == "prod" do
+  config :screenplay, Oban,
+    plugins: [
```

Comment:
We should probably also use `Pruner`. Per [the docs](https://hexdocs.pm/oban/Oban.Plugins.Pruner.html):

> Pruning is critical for maintaining table size and continued responsive job processing. It is recommended for all production applications.

It also seems like we'd want the default behaviors of `Lifeline` and `Reindexer`.

Reply:
Ahh, missed those in the docs. Thanks.